test_scripts/WebEngine/WebEngineProjection/005_Registration_disallowed_app_revoked_permissions_in_PT.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that App will be disallowed to register with HMI type WEB_VIEW
+-- when application has revoked permissions in policy table
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. PT contains record for App1 with null permissions
+--
+-- Sequence:
+-- 1. Application1 tries to register with WEB_VIEW appHMIType
+--  a. SDL rejects registration of application (resultCode: "DISALLOWED", success: false)
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultMobileAdapterType = "WS"
+
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = { "WEB_VIEW" }
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = appHMIType
+
+--[[ Local Functions ]]
+local function updatePreloadedPT(pAppId)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.null
+  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
```

Comment:
Is it necessary to invalidate the DataConsent-2 functional groupings?

Reply:
@JackLivio Please notice this is not invalidation actually.
Currently `sdl_preloaded_pt.json` has the followoing definition for `DataConsent-2` group:
```
"DataConsent-2": {
    "user_consent_prompt": "DataConsent",
    "rpcs": null
}
```
Unfortunately lua tables have no representation for `null` value. 
In order to store this value in table specific `common.null` constant is used.
Then this constant is also used when lua table is converted back into `json`.